apps/web/src/screens/Profile/modals/Passcode/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC, useState } from 'react';
+import { ModalConfigs, PasscodeFeature } from '@walless/app';
+import { Text } from '@walless/gui';
+import { ResponseCode } from '@walless/messaging';
+import {
+	PendingTransactionContext,
+	transactionActions,
+} from 'state/transaction';
+
+import ModalWrapper from '../components/ModalWrapper';
+
+const logoUri = { uri: '/img/bare-icon.png' };
```

Comment:
create `utils/config.ts` that manage all assets resources.. and then import assets (object that hold uri) from there.